plugin/pkg/skplug/dispatcher/dispatcher.go

```diff
@@ -0,0 +1,128 @@
+package dispatcher
+
+import (
+	"fmt"
+	"github.com/hashicorp/go-plugin"
+	"github.com/josephburnett/sk-plugin/pkg/skplug"
+	"github.com/josephburnett/sk-plugin/pkg/skplug/proto"
+	"os"
+	"os/exec"
+	"sync"
+)
+
+type Dispatcher interface {
+	Init(pluginsPaths []string)
```

Comment:
What if we want to pass arguments to a plugin?  E.g. command line flags.  Should this be `pluginCommands` or the like?

No action required.  Just wondering if you've through about this.

Reply:
Yeah, then we should consider that we have pluginPath and pluginCommands. But we need to give more thought how to structure it.

Reply:
Feel free to ignore for now.